### PR TITLE
change-p

### PR DESCRIPTION
**Pull request by AnhelinaV**
documented p for blocks 3 and 5, changed it for block 7, as these p rewrite block 4

12/10 16:18 - deleted comment code

### CODE REVIEW

File: block3/block3.css [hunk 1]
```diff
@@ -27,13 +27,14 @@ h1 {
     color: rgb(0, 0, 0); 
  } 
 
-p{
+/* p{
```
Comment (lapc002) [inline block3/block3.css]: please remove block instead of comment it. It makes code more clear

File: block5/block5.css [hunk 2]
```diff
@@ -20,12 +20,14 @@ h1 {
   margin-bottom: 40px;
   font-family: "Source Sans Pro", sans-serif;
 }
-p {
+
+/* p {
```
Comment (lapc002) [inline block5/block5.css]: the same here

File: block3/block3.css (hunk 1 above)
Comment (AnhelinaV) [reply block3/block3.css]: done

File: block5/block5.css (hunk 2 above)
Comment (AnhelinaV) [reply block5/block5.css]: done

**Pull request merged**